WordPress/Classes/System/3DTouch/WP3DTouchShortcutCreator.swift

```diff
@@ -26,15 +26,23 @@ public class WP3DTouchShortcutCreator: NSObject
         blogService = BlogService(managedObjectContext: mainContext)
     }
     
-    public func createShortcuts(loggedIn: Bool) {
+    public func createShortcutsIf3DTouchAvailable(loggedIn: Bool) {
+        if !is3DTouchAvailable() {
+            return
+        }
```

Comment:
It's a good practice to use `guard` for this. The resulting behavior would be the same, but `guard` shows a clearer intention, and the compiler will warn you if you forget to add a return, throw, or similar inside the `guard`